drivers/sdhc/rcar_mmc_registers.h

```diff
@@ -12,210 +12,209 @@
+#define RCAR_MMC_OPTION_WIDTH_4	   (0 << 13)
+#define RCAR_MMC_OPTION_WIDTH_8	   (1 << 13)
 
 /*
  * The SD error status register 1 indicates the CRC status, CRC error,
  * End error, and CMD error.
  */
-#define RCAR_MMC_ERR_STS1		0x0B0
+#define RCAR_MMC_ERR_STS1 0x0B0
 
 /* The SD error status register 2 indicates the timeout state. */
-#define RCAR_MMC_ERR_STS2		0x0B8
+#define RCAR_MMC_ERR_STS2 0x0B8
 
 /* SD Buffer Read/Write Register */
-#define RCAR_MMC_BUF0			0x0C0
+#define RCAR_MMC_BUF0 0x0C0
 
 /* The DMA mode enable register enables the DMA transfer. */
-#define RCAR_MMC_EXTMODE		0x360
-#define RCAR_MMC_EXTMODE_DMA_EN		BIT(1)	/* transfer 1: DMA, 0: pio */
+#define RCAR_MMC_EXTMODE	0x360
+#define RCAR_MMC_EXTMODE_DMA_EN BIT(1) /* transfer 1: DMA, 0: pio */
 
 /* The software reset register sets a software reset. */
-#define RCAR_MMC_SOFT_RST		0x380
-#define RCAR_MMC_SOFT_RST_RSTX		BIT(0)	/* reset deassert */
+#define RCAR_MMC_SOFT_RST      0x380
+#define RCAR_MMC_SOFT_RST_RSTX BIT(0) /* reset deassert */
 
 /* The version register indicates the version of the SD host interface. */
-#define RCAR_MMC_VERSION		0x388
-#define RCAR_MMC_VERSION_IP		0xff	/* IP version */
+#define RCAR_MMC_VERSION    0x388
+#define RCAR_MMC_VERSION_IP 0xff /* IP version */
 
 /*
  * The host interface mode setting register selects the width for access to
  * the data bus.
  */
-#define RCAR_MMC_HOST_MODE		0x390
+#define RCAR_MMC_HOST_MODE 0x390
 
 /* The SD interface mode setting register specifies HS400 mode. */
-#define RCAR_MMC_IF_MODE		0x398
-#define RCAR_MMC_IF_MODE_DDR		BIT(0)	/* DDR mode */
+#define RCAR_MMC_IF_MODE     0x398
+#define RCAR_MMC_IF_MODE_DDR BIT(0) /* DDR mode */
 
 /* Set of DMAC registers */
-#define RCAR_MMC_DMA_MODE		0x820
-#define RCAR_MMC_DMA_MODE_DIR_RD	BIT(16)	/* 1: from device, 0: to dev */
-#define RCAR_MMC_DMA_MODE_WIDTH		(BIT(4) | BIT(5))
-#define RCAR_MMC_DMA_MODE_ADDR_INC	BIT(0)	/* 1: address inc, 0: fixed */
-#define RCAR_MMC_DMA_CTL		0x828
-#define RCAR_MMC_DMA_CTL_START		BIT(0)	/* start DMA (auto cleared) */
-#define RCAR_MMC_DMA_RST		0x830
-#define RCAR_MMC_DMA_RST_DTRAN0		BIT(8)
-#define RCAR_MMC_DMA_RST_DTRAN1		BIT(9)
-#define RCAR_MMC_DMA_INFO1		0x840
-#define RCAR_MMC_DMA_INFO1_END_RD2	BIT(20)	/* DMA from device is complete (uniphier) */
-#define RCAR_MMC_DMA_INFO1_END_RD	BIT(17)	/* DMA from device is complete (renesas) */
-#define RCAR_MMC_DMA_INFO1_END_WR	BIT(16)	/* DMA to device is complete */
-#define RCAR_MMC_DMA_INFO1_MASK		0x848
-#define RCAR_MMC_DMA_INFO2		0x850
-#define RCAR_MMC_DMA_INFO2_ERR_RD	BIT(17)
-#define RCAR_MMC_DMA_INFO2_ERR_WR	BIT(16)
-#define RCAR_MMC_DMA_INFO2_MASK		0x858
-#define RCAR_MMC_DMA_ADDR_L		0x880
-#define RCAR_MMC_DMA_ADDR_H		0x888
+#define RCAR_MMC_DMA_MODE	   0x820
+#define RCAR_MMC_DMA_MODE_DIR_RD   BIT(16) /* 1: from device, 0: to dev */
+#define RCAR_MMC_DMA_MODE_WIDTH	   (BIT(4) | BIT(5))
+#define RCAR_MMC_DMA_MODE_ADDR_INC BIT(0) /* 1: address inc, 0: fixed */
```

Comment:
Could you please make a small experiment for me?

Create another define with a longer name, like 

`#define RCAR_MMC_DMA_MODE_ADDR_INC_2 BIT(0) `

and run this tool again. Will in rearrange the whole block again?

Reply:
[longer_name.patch.txt](https://github.com/xen-troops/zephyr/files/11642116/longer_name.patch.txt)


maybe it is better don't touch this file...

Reply:
I've dropped changes in this file
I think prev formatting is better

Reply:
Thank you. I was afraid that it will do exactly this.

The same thing is going with `#define MMC_POLL_FLAGS_TIMEOUT_US` and its neighbours. This is the case I told you about, when asked to not align `#define` constants.